src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -70,10 +71,11 @@ class PropertyAccessor implements PropertyAccessorInterface
      * Should not be used by application code. Use
      * {@link PropertyAccess::createPropertyAccessor()} instead.
      */
-    public function __construct(bool $magicCall = false, bool $throwExceptionOnInvalidIndex = false, CacheItemPoolInterface $cacheItemPool = null)
+    public function __construct(bool $magicCall = false, bool $throwExceptionOnInvalidIndex = false, bool $throwExceptionOnInvalidPropertyPath = true, CacheItemPoolInterface $cacheItemPool = null)
```

Comment:
that's a hard BC break, we don't accept them adding the argument last could work

Reply:
Got it. Will change arguments  order.